shared/stateutil/state_root_test.go

```diff
@@ -9,11 +9,19 @@ import (
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
 	"github.com/prysmaticlabs/go-ssz"
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
 	"github.com/prysmaticlabs/prysm/shared/interop"
 	"github.com/prysmaticlabs/prysm/shared/params"
 	"github.com/prysmaticlabs/prysm/shared/stateutil"
 )
 
+func init() {
+	config := &featureconfig.Flags{
+		EnableSSZCache: true,
+	}
+	featureconfig.Init(config)
+}
+
```

Comment:
Is this being used? Hard to tell.

Reply:
Yep! It is because it is picked up by the internal stateutil.HashTreeRoot function as being enabled